### PR TITLE
Add Relay section to Products page

### PR DESCRIPTION
**Pull request by reemhamz**
## Description
The Products page was missing a Firefox Relay section, so added that in.
## Issue / Bugzilla link
#10709 
## Testing
http://localhost:8000/en-US/firefox/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/home/index-master.html [hunk 1]
```diff
@@ -223,6 +223,19 @@ <h5 class="mzp-c-menu-list-title">{{ ftl('firefox-home-download-the-app') }}</h5
...
+      image_url='img/firefox/home/master/firefox-relay.jpg',
+      block_class='t-monitor mzp-l-split-reversed',
+      media_class='mzp-l-split-h-center',
+      media_after=True
+    ) %}
+      <div class="mzp-c-wordmark mzp-t-wordmark-md mzp-t-product-relay">
+        <h3>{{ ftl('firefox-home-firefox-relay') }}</h3>
+      </div>
+      <h4>{{ ftl('firefox-home-un-spam-your-life') }}</h4>
+      <a class="mzp-c-cta-link" href="{{ 'https://relay.firefox.com/' + referrals }}" data-link-name="Monitor" data-link-type="link">{{ ftl('firefox-home-try-relay') }}</a>
```
Comment (maureenlholland) [inline bedrock/firefox/templates/firefox/home/index-master.html:236]: **issue (blocking):** typo for GA
```suggestion
      <a class="mzp-c-cta-link" href="{{ 'https://relay.firefox.com/' + referrals }}" data-link-name="Relay" data-link-type="link">{{ ftl('firefox-home-try-relay') }}</a>
```

**Review by maureenlholland — CHANGES_REQUESTED** [review]
Looks nice and clean! One typo to fix.

**issue (non-blocking):** the horizontal whitespace around the image looks a little extra-indented in comparison to other product blocks

**suggestion:** is it worth trying to chop off that whitespace? might line up better with the pocket text above. 

Screen Shot 2022-03-28 at 3 25 37 PM

File: bedrock/firefox/templates/firefox/home/index-master.html (hunk 1 above)
Comment (reemhamz) [reply bedrock/firefox/templates/firefox/home/index-master.html:236]: nice catch!

**Comment by reemhamz** [comment]
Going to leave the image as-is, because when I did try to remove some/all the white space, the image ended up looking much larger compared to the other product images. Happy to leave as-is until we get a better picture to use.

Examples:
#### With original size and white space
Screen Shot 2022-03-28 at 10 58 30 AM

#### After removing white space

Screen Shot 2022-03-28 at 10 59 30 AM

**Comment by reemhamz** [comment]
@slightlyoffbeat is there a better picture (also probably a bit more high-res) that isn't sized too differently we could use for this section?

**Pull request merged**